app/helpers/application_helper/button/vm_reconfigure.rb

```diff
@@ -2,6 +2,6 @@ class ApplicationHelper::Button::VmReconfigure < ApplicationHelper::Button::Basi
   needs :@record
 
   def visible?
-    @record.reconfigurable?
+    role_allows_any?(:identifiers => %w(vm_reconfigure_cpu vm_reconfigure_memory vm_reconfigure_networks vm_reconfigure_disks vm_reconfigure_drives)) && @record.reconfigurable?
```

Comment:
Would perhaps `role_allows?(:feature => 'vm_reconfigure_all', :any => true)` be better here?

Reply:
yes, it makes sense and it actually simplifies things significantly, I can remove changes to 2 other places if I do it this way

let me fix it...